data/pathfinder/paizo/roleplaying_game/pathfinder_unchained/pu_abilities_class.lst

```diff
@@ -31,23 +31,53 @@ Wound Threshold										SORTKEY:0	CATEGORY:Special Ability	TYPE:SpecialQuality
+Legendary Armor		KEY:Legendary Gifts ~ Legendary Armor		CATEGORY:Special Ability		TYPE:ABP Legendary Gifts.SpecialQuality.Automatic Bonus Progression															MULT:YES		STACK:YES		CHOOSE:NOCHOICE										BONUS:VAR|ABP_ArmorBonus|2				DESC:Your enhancement bonus from armor attunement increases by 2. If you are attuned to a suit of armor and a shield, you can either increase both bonuses by 1 or increase one bonus by 2. The enhancement bonus on a single attuned item can't exceed +5, but you can use the excess to add special abilities to the armor or shield. You can select this legendary gift multiple times; it stacks up to +10 on any one suit of armor or shield.					SOURCEPAGE:p.156		ASPECT:ChildAbility|Legendary Gifts
+Legendary Body		KEY:Legendary Gifts ~ Legendary Body		CATEGORY:Special Ability		TYPE:ABP Legendary Gifts.SpecialQuality.Automatic Bonus Progression																																	BONUS:VAR|Pool_ABPPhysicalProwess|1			DESC:Increase your bonuses from physical prowess to +6/+6/+4. This counts as two legendary gifts.																																																										COST:2		SOURCEPAGE:p.156		ASPECT:ChildAbility|Legendary Gifts
+Legendary Body 2		KEY:Legendary Gifts ~ Legendary Body 2		CATEGORY:Special Ability		TYPE:ABP Legendary Gifts.SpecialQuality.Automatic Bonus Progression		PREABILITY:1,CATEGORY=Special Ability,Legendary Gifts ~ Legendary Body																				BONUS:VAR|Pool_ABPPhysicalProwess|1			DESC:Increase your bonuses from physical prowess to +6/+6/+6. You must already have legendary body to select this legendary gift.																																																								SOURCEPAGE:p.156		ASPECT:ChildAbility|Legendary Gifts
+Legendary Mind		KEY:Legendary Gifts ~ Legendary Mind		CATEGORY:Special Ability		TYPE:ABP Legendary Gifts.SpecialQuality.Automatic Bonus Progression																																	BONUS:VAR|Pool_ABPMentalProwess|1			DESC:Increase your bonuses from mental prowess to +6/+6/+4. This counts as two legendary gifts.																																																											COST:2		SOURCEPAGE:p.157		ASPECT:ChildAbility|Legendary Gifts
+Legendary Mind 2		KEY:Legendary Gifts ~ Legendary Mind 2		CATEGORY:Special Ability		TYPE:ABP Legendary Gifts.SpecialQuality.Automatic Bonus Progression		PREABILITY:1,CATEGORY=Special Ability,Legendary Gifts ~ Legendary Mind																				BONUS:VAR|Pool_ABPMentalProwess|1			DESC:Increase your bonuses from mental prowess to +6/+6/+6. You must already have legendary mind to select this legendary gift.																																																								SOURCEPAGE:p.157		ASPECT:ChildAbility|Legendary Gifts
```

Comment:
Why are we breaking apart the abilities? Can we not have a choice ability we take twice and have the desc update to reflect?

Reply:
Because that's literally how it's printed in the book.